src/data/finset/basic.lean

```diff
@@ -1416,6 +1416,15 @@ ext $ λ x, ⟨λ h, (mem_filter.1 h).1, λ hx, mem_filter.2 ⟨hx, h x hx⟩⟩
 lemma filter_false_of_mem {s : finset α} (h : ∀ x ∈ s, ¬ p x) : s.filter p = ∅ :=
 eq_empty_of_forall_not_mem (by simpa)
 
+lemma filter_false_iff (s : finset α) (p : α → Prop) [decidable_pred p] :
```

Comment:
```suggestion
lemma filter_eq_empty_iff (s : finset α) (p : α → Prop) [decidable_pred p] :
```

Reply:
That's (almost) what I originally called it, and I only changed it to match the existing lemmas like `filter_true_of_mem` and `filter_false_of_mem`